src/Functional.CQS.AOP.IoC.SimpleInjector.Caching/Functional.CQS.AOP.IoC.SimpleInjector.Caching.csproj

```diff
@@ -9,16 +9,17 @@
 
 Used with SimpleInjector container.</Description>
     <Copyright>2019</Copyright>
-    <PackageLicenseUrl>https://github.com/RyanMarcotte/Functional.CQS/blob/master/LICENSE</PackageLicenseUrl>
     <PackageProjectUrl>https://github.com/RyanMarcotte/Functional.CQS</PackageProjectUrl>
     <PackageTags>functional CQS netstandard netcore simpleinjector aop ioc caching</PackageTags>
-    <Version>1.0.1</Version>
+    <Version>2.00</Version>
```

Comment:
Did you miss a `.` to make it `2.0.0`?

Reply:
Indeed I did.  I'll fix that before I push the packages out later tonight.